projects/plugins/crm/api/customers.php

```diff
@@ -65,6 +67,11 @@
 	$companyID = (int) $customer_params['company'];
 }
 
+$withTags = false;
+if ( isset( $customer_params['tags'] ) ) {
+	$withTags = true;
+}
```

Comment:
Technically, if someone provides "false" as the parameter value, this would still include tags because we just look for its existence
```suggestion
$withTags = false;
if ( isset( $customer_params['tags'] ) && $customer_params['tags'] ) {
	$withTags = true;
}
```

Reply:
Yes! It'll work with any value distinct to `0`, `false` or `null`

Also, this way works too:

```
if ( isset( $customer_params['tags'] ) ) {
	$withTags = !! $customer_params['tags'];
}
```

Reply:
Either solution is 💯 with me as long as we respect "falsy values" since that's how engineers would probably understand it 😄